### DISCUSSION TITLE
Kamon and java grpc

### DISCUSSION THREAD

**Discussion by Tochemey** [kamon-io/Kamon#885] [Q&A] [answered]
I would like to know whether Kamon can help in instrumenting a java grpc application aside an akka based application. 
I have seen this repo https://github.com/kamon-io/kamon-netty and since java grpc is based upon Netty it may do the job.

**Comment by SimunKaracic** [comment, marked as answer]
We don't currently support it, but we plan to add support sometime in the future

**Reply by Tochemey** [reply]
Ok cool. Hope to see happening because the whole ecosystem in term of tracing and metrics in the JVM ecosystem is just not easy for any engineer to quickly hit the ground running.
